lxd/identities.go

```diff
@@ -128,6 +144,43 @@ var oidcIdentityCmd = APIEndpoint{
 	},
 }
 
+var bearerIdentityCmd = APIEndpoint{
+	Name:        "identity",
+	Path:        "auth/identities/bearer/{nameOrIdentifier}",
+	MetricsType: entity.TypeIdentity,
+
+	Get: APIEndpointAction{
+		Handler:       getIdentity,
```

Comment:
The handler names are not aligned with the convention of `<entity><verb>` of the rest of the access handler names.

Reply:
Now I look closer it seems that the identities endpoints have all gone their own way with the naming convention. We should bring them back inline with the rest of the project.